OrrRock/OrrRock/ViewControllers/HomeViewController/HomeViewController+CollectionViewExtensions.swift

```diff
@@ -20,37 +20,75 @@ extension HomeViewController: UICollectionViewDataSource {
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         
-        var listCell : HomeCollectionViewListCell!
-        var cardCell : HomeCollectionViewCardCell!
-        
         if isCardView {
-            cardCell = collectionView.dequeueReusableCell(withReuseIdentifier: "homeCollectionViewCardCell", for: indexPath) as? HomeCollectionViewCardCell
-            cardCell.detailButton.tag = indexPath.row
-            cardCell.detailButton.addTarget(self, action:  #selector(navigateToVideoCollectionView(sender:)), for: .touchUpInside)
-            return cardCell
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "homeCollectionViewCardCell", for: indexPath) as! HomeCollectionViewCardCell
+            
+            var successCount: Int = 0
+            var thumbnails: [UIImage] = []
+            
+            let primaryTitle: String = sortOption == .gymVisitDate ? sortedVideoInfoData[indexPath.row][0].gymVisitDate.timeToString() : sortedVideoInfoData[indexPath.row][0].gymName
+            
+            let secondaryTitle: String = sortOption == .gymVisitDate ? sortedVideoInfoData[indexPath.row][0].gymName : "\(min(sortedVideoInfoData[indexPath.row].first!.gymVisitDate, sortedVideoInfoData[indexPath.row].last!.gymVisitDate).timeToString()) ~  \(max(sortedVideoInfoData[indexPath.row].first!.gymVisitDate, sortedVideoInfoData[indexPath.row].last!.gymVisitDate).timeToString())"
```

Comment:
삼항연산자에, 문자열보간에, 너무 코드가 길어져서
혹시 이런 방식은 어떠신가 조심스럽게 제안해 봅니다~

```suggestion
	    var sortedFirstData = sortedVideoInfoData[indexPath.row][0]
	    var firstVisitDate = sortedVideoInfoData[indexPath.row].first!.gymVisitDate
	    var lastVisitDate = sortedVideoInfoData[indexPath.row].last!.gymVisitDate
	
	    let primaryTitle: String = sortOption == .gymVisitDate ? sortedFirstData.gymVisitDate.timeToString() : sortedFirstData.gymName
	    let secondaryTitle: String = sortOption == .gymVisitDate ? sortedFirstData.gymName : "\(min(firstVisitDate, lastVisitDate).timeToString()) ~  \(max(firstVisitDate, lastVisitDate).timeToString())"
```